packages/lesswrong/components/tagging/FilterMode.tsx

```diff
@@ -235,7 +246,8 @@ function filterModeToStr(mode: FilterMode, currentUser: UsersCurrent | null): st
       Math.abs(0.5 - mode) < .000000001 &&
       userHasNewTagSubscriptions(currentUser)
     ) return "Reduced"
-    if (mode > 0) return `+${mode}`
+    if (mode >= 1) return `+${mode}`
+    if (mode > 0) return `-${Math.round((1 - mode) * 100)}%`
     if (mode === 0) return ""
```

Comment:
I relatedly can't get to .55, because it captures me at .5 and redirects me to "Reduced"

Reply:
(I already left this comment but it isn't showing up for me, so ignore if this is a duplicate)

I've just pushed a pretty big change to the logic here where it'll now wait for 1.5 seconds after entering a preset value before clearing the input. Opinions are very welcome - I think I like the UX, but I could probably be convinced otherwise. This is similar to how vim works, but maybe vim isn't the best place to get UX ideas.